### PR TITLE
Show collapsed ellipsis again

### PR DESCRIPTION
**Pull request by jerone**
Small fix that shows the ellipsis (`...`) again when object is collapsed.

Great addon btw!

### CODE REVIEW

**Comment by jerone** [comment]
Anything wrong with this PR?

**Comment by TrangPham** [comment]
Connected to #89
(So that the issue also links back)

**Comment by jerone** [comment]
Any issues with this PR that blocks merging?

File: data/default.css [hunk 1]
```diff
@@ -45,7 +45,7 @@ body {
 }
 
 .collapsible.collapsed {
-  height: .8em;
+  height: .9em;
```
Comment (stanio) [inline data/default.css]: This doesn't appear to help me (#96) on Windows, and actually tried the other day on Mac, unless I use `height: 1em`.

Comment (jerone) [inline data/default.css]: `0.9em` is minimum for me on Windows. Tried `1em` and that works too. Still weird, do you have a theme installed, or zoomed in?

Comment (stanio) [inline data/default.css]: On Windows I'm on a high DPI (200% zoom) screen.  Anyway, I don't think anything below `1em` should be considered without risk of obstruction.  In the given case where the ellipsis glyph appears at the bottom of the character box – it is very likely it could be hidden given a box of no full character height.

Comment (stanio) [inline data/default.css]: > On Windows I'm on a high DPI (200% zoom) screen.

Looking at the screenshots on a low res now - I see even with `1em` box height the ellipsis are just a tiny bit cut at the bottom.

> Anyway, I don't think anything below `1em` should be considered without risk of obstruction...

The perfect solution would be to set the box height to the font `normal` line-height (which varies between `1` and `1.2` often), but I don't think is currently possible in CSS:

http://lists.w3.org/Archives/Public/www-style/2003Sep/0120.html

Comment (stanio) [inline data/default.css]: See what I'm talking about:

*   `height: .8em` (original)

    height: 0.8em

*   `height: .9em`

    height: 0.9em

    _Doesn't appear to solve the problem._

*    `height: 1em`

    height: 1em

    _Partially solves the problem._

*    `height: 1.2em`

    height: 1.2em

    _Appears to solve the problem (see reasoning in my previous comment)._

Comment (jerone) [inline data/default.css]: Send a PR with above screensies.

Comment (bhollis) [inline data/default.css]: I would accept a pull request with these changes.

Comment (stanio) [inline data/default.css]: > Send another PR with above screensies.

I've previously opened #96 - it was closed without being properly addressed.  I guess I've spent too much time and resources on this, if others doesn't appear interested in it.

Comment (jerone) [inline data/default.css]: [**@stanio**](https://github.com/stanio) commented on [15 mei 2015 10:49 CEST](https://github.com/bhollis/jsonview/pull/83/files#r30393663 "2015-05-15T08:49:29Z - Replied by Github Comment Enhancer"):
> > Send another PR with above screensies.
> 
> I've previously opened [#96](https://github.com/bhollis/jsonview/issues/96 "Ellipsis for collapsed object/array not visible") - it was closed without being properly addressed.  I guess I've spent too much time and resources on this, if others doesn't appear interested in it.

#96 isn't a PR. @bhollis already mentioned that he will accept a PR. I even [confirmed](https://github.com/bhollis/jsonview/pull/83/files#r29788055) that your fix works on my system. Just send another PR, because this is already merged.
